include/alpaka/mem/buf/omp5/Set.hpp

```diff
@@ -96,20 +97,23 @@ namespace alpaka
                 std::uint8_t const& byte,
                 TExtent const& /* extent */)
             {
-                using Idx = typename alpaka::trait::IdxType<TExtent>::type;
-                using Dim1D = DimInt<1u>;
-                using Vec1D = Vec<Dim1D, Idx>;
-
                 static_assert(Dim<TView>::value == 0u, "The view is required to have dimensionality 0!");
                 static_assert(Dim<TExtent>::value == 0u, "The extent is required to have dimensionality 0!");
 
-                return createTaskKernel<AccOmp5<Dim1D, Idx>>(
-                    WorkDivMembers<Dim1D, Idx>(Vec1D::zeros(), Vec1D::zeros(), Vec1D::zeros()),
-                    MemSetKernel(),
+                using ExtIdx = Idx<TExtent>;
+                using Dim0 = DimInt<0u>;
```

Comment:
Why do you change this kernel? This kernel must be a one-dimensional kernel to emulate the memset function on the device.

Reply:
AFAIK I only changed the implementation of the zero dimensional Set implementation, where there are no pitches and we only set `sizeof(TElem)` bytes. I can add a comment to point that out more explicitely.